app/medInria/medVisualizationWorkspace.h

```diff
@@ -27,11 +27,9 @@ class medVisualizationWorkspace : public medAbstractWorkspace
                             "Workspace to visualise images, meshes and other data types.",
                             "Basic")
 public:
-     medVisualizationWorkspace(QWidget *parent = 0);
+     medVisualizationWorkspace(QWidget *parent = 0){}
```

Comment:
Also, ```d``` is not being initialised anymore. If it is not used it should be removed.